consumer/src/main/java/no/nav/sbl/dialogarena/modiabrukerdialog/consumer/service/saker/mediation/SakApiGateway.kt

```diff
@@ -0,0 +1,89 @@
+package no.nav.sbl.dialogarena.modiabrukerdialog.consumer.service.saker.mediation
+
+import com.fasterxml.jackson.datatype.joda.JodaModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.common.log.MDCConstants
+import no.nav.common.rest.client.RestClient
+import no.nav.common.sts.SystemUserTokenProvider
+import no.nav.common.utils.EnvironmentUtils
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.utils.RestConstants
+import no.nav.sbl.dialogarena.modiabrukerdialog.api.utils.TjenestekallLogger
+import no.nav.sbl.dialogarena.modiabrukerdialog.sak.service.FodselnummerAktorService
+import okhttp3.Request
+import okhttp3.Response
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+import java.util.*
+
+
+internal interface SakApiGateway {
+    fun hentSaker(fnr: String): List<SakDto>
+}
+
+internal class SakApiGatewayImpl(val fodselnummerAktorService: FodselnummerAktorService,
```

Comment:
Vi har typisk kalt denne typen klasser med `Client`/`ClientImpl` suffix. Så kjekt om vi kan videreføre den konvensjonen imho

Reply:
Jeg tror at det egentlig gateway pattern som vi bruker her https://martinfowler.com/eaaCatalog/gateway.html 
An object that encapsulates access to an external system or resource.